.github/workflows/on-tag.yml

```diff
@@ -0,0 +1,146 @@
+name: Build & deploy ElectrumX
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  build:
+    name: Build ElectrumX
+    runs-on: ubuntu-18.04
+
+    # NOTE: qemu v3.1.1 used instead of currently newest v4.1.0, because v4 is **much** slower for aarch64, see:
+    #   https://github.com/meeDamian/docker-berkeleydb/commit/9e87d11314c2522726497f0c6059e61a31298e7f/checks
+    env:
+      QEMU_VERSION: v3.1.1
+      DOCKER_BUILDKIT: 1
+
+    strategy:
+      matrix:
+        arch:
+          - arm32v7
+          - arm64
+          - amd64
+
+    steps:
+      - uses: actions/checkout@v1.0.0
+
+      - name: Setup environment
+        run: |
+          VERSION="$(echo "${GITHUB_REF}" | awk -F/ '{print $NF}' | tr -d v)"
+          echo ::set-env name=VERSION::"${VERSION}"
+          echo ::set-env name=DIR::"$(echo "${VERSION}" | cut -d. -f-2)"
+      - name: Register self-compiled qemu
+        if: matrix.arch != 'amd64'
+        run: docker run --rm --privileged "meedamian/simple-qemu:${QEMU_VERSION}-${{matrix.arch}}" -p yes
+
+      # Alter `Dockerfile` to reference used architecture/image combos explicitly.  Places changed are:
+      #   * all `FROM` statements (ex. `FROM alpine…` -> `FROM arm64v8/alpine…`)
+      # `sed` `--expression`s change it in the following way:
+      #   Matches all occurrences of `FROM alpine`, and injects arch prefix before `alpine`, ex: `arm64v8/alpine`
+      - name: Change Dockerfile to use arch-specific base images
+        if: matrix.arch != 'amd64'
+        run: |
+          CPU="${{matrix.arch}}"
+          if [[ "${CPU}" == "arm64" ]]; then
+            CPU="arm64v8"
+          fi
+          sed -i Dockerfile \
+            -e "s|^FROM alpine|FROM $CPU/alpine|g"
```

Comment:
This line does nothing, and this line is essential to getting it to work. You use a different base for your image, and it doesn't match `sed`'s pattern.

Reply:
wow, thank you, very good catch, now as you mention it makes totally sense, so the docker file for each stays with the same alpine arch (amd64) like the original Dockerfile and therefore we built always the same arch 🤦‍♂

I will check how those python specific arches are named and play with that regex to make it work, thank you!

Reply:
Quite sure it's: `{arm32v6,arm32v7,arm64v8,amd64}/python:3.7-alpine3.9`

Reply:
Yes found that out and adjusted regex to replace everything after "FROM " and prepend $CPU/ and it works for the arch.

However, new problem, `rocksdb-dev` (leveldb-dev works though) is not available for `arm` arches on alpinelinux it seems, so it fails there now: https://pkgs.alpinelinux.org/packages?name=rocksdb-dev&branch=edge

Next step, figure out how to build multi arch packages for Alpinelinux? 😬 https://wiki.alpinelinux.org/wiki/Creating_an_Alpine_package

I tried also install `rocksdb-dev` via pip3 on rpi4 on Debian Buster, while it finds the package it errors on building it, but OT here, just wanted to check if it is generally available elsewhere.
```
 rocksdb/_rocksdb.cpp:622:10: fatal error: rocksdb/slice.h: No such file or directory
     #include "rocksdb/slice.h"
              ^~~~~~~~~~~~~~~~~
    compilation terminated.
    error: command 'arm-linux-gnueabihf-gcc' failed with exit status 1
```